src/Node/Blockcore.Node/NodeBuilder.cs

```diff
@@ -27,6 +27,9 @@ public static IFullNodeBuilder Create(string chain, NodeSettings settings)
                     break;
                 case "CITY":
                 case "STRAT":
+                case "RUTA":
+                    nodeBuilder.UsePosConsensus().AddPowPosMining().UseColdStakingWallet();
```

Comment:
This like is not needed, it can fall through to the bottom which is exactly the same.

Reply:
Thanks @sondreb !! We have removed it.